src/main/java/com/exadel/sandbox/officeFloorAttachment/repositories/AttachmentRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.exadel.sandbox.officeFloorAttachment.repositories;
+
+import com.exadel.sandbox.officeFloorAttachment.entities.Attachment;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@Component
```

Comment:
Please remove @Component and check other places.
Please try to find the difference between @Component, @Repository, @Service, @Controller
Maybe it helps: 
https://stackoverflow.com/questions/6827752/whats-the-difference-between-component-repository-service-annotations-in
https://www.baeldung.com/spring-component-repository-service